### ISSUE TITLE
使用的镜像为ubuntu22.04-cuda12.8.1-py311-torch2.8.0-vllm0.11.0-modelscope1.31.0-swift3.11.0

### ISSUE THREAD

**Issue by TAOSHss** [modelscope/ms-swift#6886]
**Describe the bug**

在grpo的训练过程中，在wandb的展示界面中我看到有这个，runs.summary["completions"]应该是可以在下面展示，可以看回答效果的；但是现在是看不到的，上面是我的completions的回复长度，然后训练了几百个step了

<img width="1294" height="761" alt="Image" src="https://github.com/user-attachments/assets/ccfa183a-5620-4907-8014-ffafcc40036a" />

**Your hardware and system info**
ubuntu22.04-cuda12.8.1-py311-torch2.8.0-vllm0.11.0-modelscope1.31.0-swift3.11.0 使用的镜像版本是这个，我写在卸载了 原来的swift3.10.0，从github安装的3.11.0

**Additional context**
Add any other context about the problem here(在这里补充其他信息)

**Comment by hjh0119** [comment]
plz provide reproduce script

**Comment by TAOSHss** [comment]
> plz provide reproduce script
哦？我试试

**Comment by TAOSHss** [comment]
> plz provide reproduce script

截图看看呢，我没搜到plz

**Comment by TAOSHss** [comment]
> plz provide reproduce script

CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \
NPROC_PER_NODE=8 \
PYTORCH_CUDA_ALLOC_CONF='expandable_segments:True' \
megatron rlhf \
    --rlhf_type grpo \
    --model Qwen/Qwen3-30B-A3B-Instruct-2507 \
    --load_safetensors true \
    --save_safetensors true \
    --context_parallel_size 1 \
    --tensor_model_parallel_size 4 \
    --expert_model_parallel_size 4 \
    --pipeline_model_parallel_size 2 \
    --dataset open-r1/DAPO-Math-17k-Processed \
    --max_epochs 1 \
    --global_batch_size 8 \
    --micro_batch_size 1 \
    --steps_per_generation 1 \
    --num_generations 8 \
    --reward_funcs accuracy format \
    --use_vllm true \
    --vllm_mode colocate \
    --vllm_gpu_memory_utilization 0.4 \
    --vllm_tensor_parallel_size 8 \
    --vllm_max_model_len 16384 \
    --max_length 8192 \
    --max_completion_length 8192 \
    --train_type full \
    --lr 1e-6 \
    --bf16 true \
    --beta 0.00 \
    --importance_sampling_level sequence \
    --epsilon 3e-4 \
    --epsilon_high 4e-4 \
    --dynamic_sample false \
    --overlong_filter true \
    --loss_type grpo \
    --sleep_level 2 \
    --offload_model true \
    --offload_bridge false \
    --offload_optimizer true \
    --optimizer_cpu_offload true \
    --use_precision_aware_optimizer \
    --log_interval 1 \
    --recompute_granularity selective \
    --finetune \
    --num_workers 8 \
    --dataset_num_proc 8 \
    --no_save_optim \
    --no_save_rng \
    --attention_backend flash \
    --temperature 1.0 \
    --padding_free true \
    --sequence_parallel true \
    --log_completions true \
    --wandb_project megatron_swift \
    --wandb_exp_name megatron_grpo \

**Comment by hjh0119** [comment]
run normally on my side, please try refreshing the page.

<img width="3284" height="1564" alt="Image" src="https://github.com/user-attachments/assets/1b829a3a-1511-4b52-bd53-6b73a29108c3" />

**Comment by z0o0ey** [comment]
please retry with latest swift and vllm, feel free to reopen or create a new issue if it the problem persists.
